k2/python/k2/fsa.py

```diff
@@ -252,6 +251,41 @@ def __str__(self) -> str:
 
         return ans
 
+
+    def draw(self, filename: Optional[str], title: Optional[str] = None) -> 'Digraph':
```

Comment:
It should be `filename: Optional[str] = None`, otherwise it will crash when called with no arguments.